custom_metric/worker.py

```diff
@@ -0,0 +1,97 @@
+import asyncio
+import time
+from concurrent.futures import ThreadPoolExecutor
+from datetime import timedelta
+
+from temporalio import activity, workflow
+from temporalio.client import Client
+from temporalio.runtime import PrometheusConfig, Runtime, TelemetryConfig
+from temporalio.worker import (
+    ActivityInboundInterceptor,
+    ExecuteActivityInput,
+    Interceptor,
+    Worker,
+)
+
+
+@activity.defn
+def print_message():
+    print("In the activity.")
+    time.sleep(1)
+
+
+@workflow.defn
```

Comment:
Pedantic, but can you move the workflow to its own file (and pass through the import used to reference the activity)? We have issues currently where users are reading our samples and thinking their workflows should be in the same file with other things.